### PR TITLE
Respond to feedback on cluster template docs

### PR DESCRIPTION
**Pull request by catherineluse**
This PR is a follow up to the feedback on this other PR: https://github.com/rancher/docs/pull/3482/files
https://github.com/rancher/docs/issues/3258

### CODE REVIEW

File: content/rancher/v2.6/en/admin-settings/cluster-templates/_index.md [hunk 1]
```diff
@@ -54,15 +59,34 @@ The example repository for an RKE2 cluster template is [here](https://github.com
...
+# Deploying Clusters from a Template with Fleet
+
+> **Prerequisites:**
+>
+> - You will need permission to provision new Kubernetes clusters.
+> - You will need cloud credentials for provisioning infrastructure using the template.
+> - In order to show in the form for creating new clusters, the cluster template's Helm chart must have the `catalog.cattle.io/type:cluster-template` annotation.
+> - In order to use a template as part of continuous delivery/GitOps, the cluster template needs to be deployed in the `fleet-local` namespace of the `local` cluster.
+> - All values must be set in the `values.yaml` of the template.
+
```
Comment (slickwarren) [inline content/rancher/v2.6/en/admin-settings/cluster-templates/_index.md:81]: one more prerequisite: fleet repos must follow a separate guideline: http://fleet.rancher.io/gitrepo-structure/. For rke2 cluster templates, that means adding a fleet.yaml file to their repo.

File: content/rancher/v2.6/en/admin-settings/cluster-templates/_index.md [hunk 2]
```diff
@@ -54,15 +59,34 @@ The example repository for an RKE2 cluster template is [here](https://github.com
...
 1. Click **☰ > Cluster Management**.
 1. On the **Clusters** page, click **Create.**
 1. Click **Create Cluster from Template.**
 
-**Result:** After Rancher provisions the new cluster, it is managed in the same way as any other Rancher-launched Kubernetes cluster.
+**Result:** After Rancher provisions the new cluster, it is managed by Fleet. You can configure any options through the UI if the cluster template has options for the user to choose from.
```
Comment (slickwarren) [inline content/rancher/v2.6/en/admin-settings/cluster-templates/_index.md:86]: `You can configure any options through the UI if the cluster template has options for the user to choose from.` This is not possible for fleet, afaik. This is why the values.yaml must all be configured in the repo, as the user can't manipulate them when installed with fleet in rancher.

File: content/rancher/v2.6/en/admin-settings/cluster-templates/_index.md [hunk 3]
```diff
@@ -54,15 +59,34 @@ The example repository for an RKE2 cluster template is [here](https://github.com
...
+
+An admin with access to the `local` cluster can also remove a cluster deployed via cluster templates through the **Apps & Marketplace > Installed Apps** page. This user can also update any clusters using a template from this menu, given there is a new version of a template being used by those clusters.
```
Comment (slickwarren) [inline content/rancher/v2.6/en/admin-settings/cluster-templates/_index.md:88]: I think this part belongs in the section below; this is another way to uninstall cluster templates.

**Review by slickwarren — CHANGES_REQUESTED** [review]

File: content/rancher/v2.6/en/admin-settings/cluster-templates/_index.md (hunk 1 above)
Comment (catherineluse) [reply content/rancher/v2.6/en/admin-settings/cluster-templates/_index.md:81]: updated

File: content/rancher/v2.6/en/admin-settings/cluster-templates/_index.md (hunk 2 above)
Comment (catherineluse) [reply content/rancher/v2.6/en/admin-settings/cluster-templates/_index.md:86]: updated

File: content/rancher/v2.6/en/admin-settings/cluster-templates/_index.md (hunk 3 above)
Comment (catherineluse) [reply content/rancher/v2.6/en/admin-settings/cluster-templates/_index.md:88]: updated

**Review by slickwarren — APPROVED** [review]

**Pull request merged**